.github/workflows/test.yaml

```diff
@@ -87,18 +87,18 @@ jobs:
           chmod +x plural
           cp plural /usr/local/bin/plural
       - run: make testup
-      - name: Restore dependencies cache
-        uses: actions/cache@v3
-        with:
-          path: deps
-          key: ${{ runner.os }}-mix-5-${{ hashFiles('**/mix.lock') }}
-          restore-keys: ${{ runner.os }}-mix-5
-      - name: Restore _build
-        uses: actions/cache@v3
-        with:
-          path: _build
-          key: ${{ runner.os }}-mix-5-${{ hashFiles('**/mix.lock') }}
-          restore-keys: ${{ runner.os }}-mix-5
+      # - name: Restore dependencies cache
```

Comment:
we should keep the dependencies cache for sure, it likely just needs the cache key to be bumped

Reply:
Yeah that is why I disabled for now, since the PR was just being used for the tests up to this point. We could probably also be a bit smarter in terms of the key naming, like if we use the elixir and/or erlang versions instead we could build/test against multiple version with cache more easily and it would be clear what cache key to use.